jms/README.md

```diff
@@ -3,25 +3,85 @@
 This document defines how to describe JMS-specific information on AsyncAPI.
 
 <a name="version"></a>
+## Versions
 
-## Version
+The version of this bindings specification is `0.0.1`.
+This is also the `bindingVersion` for all binding objects defined by this specification.
+In any given binding object, `latest` MAY alternatively be used to refer to the currently latest published version of this bindings specification.
 
-Current version is `0.1.0`.
+## Terminology
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this bindings specification are to be interpreted as described in IETF [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Protocol
+
+These bindings use the `jms` [protocol](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#definitionsProtocol) in AsyncAPI documents to denote connections to and interactions with JMS message brokers.
+
+JMS is not technically a protocol, rather it is an API. A JMS Provider implements the JMS API and may defined a protocol for implementing JMS API operations. Regardless, for the purposes of AsyncAPI we can treat it as a "protocol" to enable AsyncAPI definitions that are somewhat portable between various JMS Providers.
+
+
+
+## Server Object
```

Comment:
do we need this section? `Server Object` is not part of the binding. I think just remove it and only provide full example of `Server Object` with binding in `Server Binding Object` section